clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -109,7 +109,7 @@
             "properties": {
                 "cobol-lsp.cpy-manager.profiles": {
                     "type": "string",
-                    "description": "Current default profile for copybook downloader"
+                    "description": "Current default profile for copybook downloader\nNOTE:Zowe Explorer version 1.15.0 or higher is required to download copybooks from the mainframe"
```

Comment:
@zeibura Please check the NOTE section.
![image](https://user-images.githubusercontent.com/69206564/148758946-db768faf-f7a5-42ba-bcfa-436a1ccc99fa.png)

Reply:
I would remove the "NOTE:"

Reply:
updated:
![image](https://user-images.githubusercontent.com/69206564/148970015-72ecc031-068b-448a-9323-66bd41124905.png)